source/frontend/src/features/leases/add/LeaseDetailSubForm.test.tsx

```diff
@@ -94,4 +111,24 @@ describe('LeaseDetailSubForm component', () => {
     expect(items[0]).toHaveTextContent(/MOCK TEST PROJECT/i);
     expect(items[1]).toHaveTextContent(/ANOTHER MOCK/i);
   });
+
+  it('displays the cancellation reason textbox whe status is changed to "Cancelled"', async () => {
```

Comment:
shouldn't there be tests for terminated as well?

Reply:
Updated